css/index.css

```diff
@@ -143,4 +144,152 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/* my code */
+
+.container .header {
+	width: 800px;
+	height: 100px;
+    display: flex;
+    justify-content: space-between;
+	align-items: center;
+}
+
+.nav {
+	width: 630px;
+    height: 96px;
+	display: flex;
+    justify-content: space-around;
+    
+}
+
+.nav-button {
+	position: relative;
+	top: 40px;
+	text-decoration: none;
+}
+
+.nav-text {
+	color: black;
+}
+
+.logo {
+    position: relative;
+    top: -10px;
+    width: 157px;
+    height: 40px;
+}
+
+#header-image {
+    width: 800px;
+    height: 155px;
+}
+
+/* BOX COLORS */
+
+#box-1 {
```

Comment:
per our discussion, look into using classes, and specificity instead of IDs when you can. remember you can assign multiple classes